File: uniffi_bindgen/src/bindings/dart/templates/UInt32Helper.py
class _UniffiConverterUInt32(_UniffiConverterPrimitiveInt) {
    CLASS_NAME = "u32"
    VALUE_MIN = 0
    VALUE_MAX = 2**32

    static read(buf) {
        return buf.read_u32()
    }

    static write_unchecked(value, buf) {
        buf.write_u32(value)
    }

}

File: uniffi_bindgen/src/bindings/dart/templates/UInt32Helper.dart
class _UniffiConverterUInt32 extends _UniffiConverterPrimitiveInt {


    int read(_UniffiRustBufferBuilder buf) {
        return buf.read_u32();
    }

    _UniffiRustBufferBuilder write(int value) {
        //buf.data.cast<Int16> = value;
        Pointer<_UniffiRustBuffer> _rustBuffer = calloc<_UniffiRustBuffer >();
        return _rustBuffer.ref.buffer;
    }
}
